DataFormats/RecoCandidate/BuildFile.xml

```diff
@@ -6,7 +6,6 @@
 <use name="DataFormats/EgammaReco"/>
 <use name="DataFormats/GsfTrackReco"/>
 <use name="clhep"/>
-<use name="SimDataFormats/TrackingAnalysis"/>
```

Comment:
apparently CommonTools/RecoAlgos/interface/CosmicTrackingParticleSelector.h (and CommonTools/RecoAlgos/BuildFile.xml ) depended on this to pull in the dependence on `SimDataFormats/TrackingAnalysis` .

So, we need to get it added to CommonTools/RecoAlgos/BuildFile.xml